lib/rules/no-hide-core-modules.js

```diff
@@ -81,8 +81,9 @@ module.exports = {
             },
         ],
         messages: {
-            "unexpectedImport": "Unexpected import of third-party module '{{name}}'.",
-        }
+            unexpectedImport:
+                "Unexpected import of third-party module '{{name}}'.",
```

Comment:
**question (non-blocking)**: Why put this on a new line?

Reply:
@scagood Huh, good catch, that must have been some automatic prettier thing, I didn't do that myself